lib/hub/commands.rb

```diff
@@ -243,11 +243,16 @@ def pull_request(args)
           msg.puts initial_message if initial_message
           msg.puts ""
           msg.puts File.read(pullrequest_template_file) if pullrequest_template_file
-          msg.puts ""
-          msg.puts "#{cc} Requesting a pull to #{base_project.owner}:#{options[:base]} from #{options[:head]}"
-          msg.puts "#{cc}"
-          msg.puts "#{cc} Write a message for this pull request. The first block"
-          msg.puts "#{cc} of text is the title and the rest is description."
+
+          if contributing_md_file
+            msg.puts File.readlines(contributing_md_file).map {|line| line.prepend('# ')}.join
+          else
+            msg.puts ""
+            msg.puts "#{cc} Requesting a pull to #{base_project.owner}:#{options[:base]} from #{options[:head]}"
```

Comment:
I think so. I mean should we append this text, even if we have a contributing.md?